include/triton/Conversion/TritonGPUToLLVM/Utility.h

```diff
@@ -1474,6 +1477,35 @@ inline bool isLayoutMmaV1(Attribute layout) {
   return isMmaV1;
 }
 
+/// @brief Extend 2d shared object to 3d.
+/// Generalizing dot operand shared to distributing conversion.
```

Comment:
"Generalizing dot operand to shared distributing conversion"?

Reply:
Yep, this part does not sound great